config/config.go

```diff
@@ -52,30 +52,22 @@ func SetDefaultDirName(binaryName string) {
 // Dir is the configured config home directory.
 // All the cli-related config files live in this directory.
 func Dir() string {
-	var dir string
+	var ev string
 	if runtime.GOOS == "windows" {
-		dir = os.Getenv("APPDATA")
-		if dir != "" {
-			return filepath.Join(dir, DefaultDirName)
-		}
+		ev = os.Getenv("APPDATA")
 	} else {
-		dir := os.Getenv("EXERCISM_CONFIG_HOME")
-		if dir != "" {
-			return dir
-		}
-		dir = os.Getenv("XDG_CONFIG_HOME")
-		if dir == "" {
-			dir = filepath.Join(os.Getenv("HOME"), ".config")
-		}
-		if dir != "" {
-			return filepath.Join(dir, DefaultDirName)
+		if ev = os.Getenv("EXERCISM_CONFIG_HOME"); ev == "" {
+			if ev = os.Getenv("XDG_CONFIG_HOME"); ev == "" {
+				ev = os.Getenv("HOME")
```

Comment:
Do we not need to append ".config" here, as in the conditional on the left in L.68?

Reply:
Hi sorry for the 1 month long delay. Yes I did append .config in the latest commit. I will mark as resolved if the result is what you expected.